<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>399</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>COFF Explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="separator"/>
    <addaction name="actionFileReload"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menu_Languages">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/images/options-languages.png</normaloff>:/images/options-languages.png</iconset>
     </property>
    </widget>
    <addaction name="menu_Languages"/>
    <addaction name="separator"/>
    <addaction name="actionOptionsOpenLastestFileAtStartup"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="separator"/>
   <addaction name="actionFileReload"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-open.png</normaloff>:/images/file-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-exit.png</normaloff>:/images/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/help-about.png</normaloff>:/images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOptionsOpenLastestFileAtStartup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open lastest file at startup</string>
   </property>
  </action>
  <action name="actionFileReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-reload.png</normaloff>:/images/file-reload.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFileOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptionsOpenLastestFileAtStartup</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileReload</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAction()</slot>
 </slots>
</ui>
